fcb-source/utilities/src/sphere_calibration.c

```diff
@@ -0,0 +1,226 @@
+/******************************************************************************
+ * @file    sphere_calibration.c
+ * @author  Dragonfly
+ * @version v. 1.0.0
+ * @date    2016-04-27
+ * @brief   Module contains a Gauss-Newton Algorithm for Sphere Fitting
```

Comment:
Perhaps we should give some mention/credit to where we got the algorithm derivation from.